src/Spectre.Service/Spectre.Service.csproj

```diff
@@ -56,11 +56,13 @@
   <ItemGroup>
```

Comment:
Please get rid of any unneccessary git-related copies of this file.

Reply:
@Daroslaw, could you be more specific?

Reply:
This is now resolved, those copies are gone